Dockerfile

```diff
@@ -38,10 +38,13 @@ FROM alpine:3.18 AS dockerize
 
 RUN apk add --no-cache openssl
 
-ENV DOCKERIZE_VERSION=v0.6.1
-RUN wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-alpine-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
-    && tar -C /usr/local/bin -xzvf dockerize-alpine-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
-    && rm dockerize-alpine-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
+ARG TARGETOS
+ARG TARGETARCH
```

Comment:
thanks for fixing this. I will merge it once docker build CI gate passes

Reply:
Yeah that should work. Let me kick off CI to validate. 
Is this change necessary  `golang:1.23.4-alpine3.21 -> golang:1.23-alpine` ?

Reply:
Not strictly. But it removes quite a few vulnerabilities from Go 1.23.4 . Reverted for now. I will open a separate pr for CVE removal.

Reply:
We need to cross check those dependency upgrades for compatibility with other integrations such as monorepo. Let's leave that Pr open for now. I will get back